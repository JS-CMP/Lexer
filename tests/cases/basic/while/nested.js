var i = 0;
while (i < 3) {
    var j = 0;
    while (j < 2) {
        j++;
    }
    i++;
}
