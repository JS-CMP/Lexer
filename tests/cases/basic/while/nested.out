#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any i;i=JS::Any(0);while (i<JS::Any(3)){JS::Any j;j=JS::Any(0);while (j<JS::Any(2)){j++;};i++;};}
